### ISSUE TITLE
Adding neuralprophet as a wrapper

### ISSUE THREAD

**Issue by aiwalter** [alan-turing-institute/sktime#529]
**Is your feature request related to a problem? Please describe.**
Can  I add `neuralprophet ` to PR #515 (which is connected to issue #503 )? I would then create a parent class for `fbprophet` and `neuralprophet` if this is fine for you @mloning ? It seems both share the same `fit()` and `oredict()` function.

**Describe the solution you'd like**
Wrapper to `neuralprophet ` https://github.com/ourownstory/neural_prophet

**Describe alternatives you've considered**
None

**Additional context**
I would like to add this once I have passed all tests with PR #515

**Comment by mloning** [comment]
Yes sounds like a good plan! We should also add it to our list of related software [here](https://www.sktime.org/en/latest/related_software.html).

**Comment by TNTran92** [comment]
Do we still need this @fkiraly? If yes, I'd like to have a shot at this

**Comment by aiwalter** [comment]
I think priority wise this hase lower priority in my opinion. And also dependency to `pytorch` might cause issues.

**Comment by TNTran92** [comment]
Thanks @aiwalter, I will pass over this one then

**Comment by vedantag17** [comment]
I would like to work on this issue. Is it still available?
